pkg/aspect/quotasManager.go

```diff
@@ -194,7 +170,10 @@ func quotaDefinitionFromProto(desc *dpb.QuotaDescriptor) (*adapter.QuotaDefiniti
 		labels[label.Name] = l
 	}
 
-	dur, _ := ptypes.DurationFromProto(desc.Expiration)
+	dur, err := ptypes.DurationFromProto(desc.Expiration)
```

Comment:
We should update the descriptors to use the gogo features such that the Go version of the descriptor would just contain a time.Duration or time.Time directly.

That's not blocking this PR, but we should do the gogo treatment on all protos where it makes sense.